be/src/exprs/jit/jit_expr.cpp

```diff
@@ -108,7 +108,7 @@ StatusOr<ColumnPtr> JITExpr::evaluate_checked(starrocks::ExprContext* context, C
     }
 #endif
 
-    auto result_column = ColumnHelper::create_column(type(), is_nullable(), is_constant(), ptr->num_rows(), false);
+    auto result_column = ColumnHelper::create_column(type(), !is_constant() && is_nullable(), is_constant(), ptr->num_rows(), false);
```

Comment:
remove `unpack_and_duplicate_const_column` in Line:98

Reply:
Done.